GetDKAN/dkan
# 2665 upgrade prefix db

Connects #2665 

**QA**
- Setup a site with prefix configuration for the db in a version of the code before this branch (ex. 7.x-1.x or 1.15.3)
- Move a few resources to the datastore
- Update the code to this branch
- Run ```drush updb```
- The datastores should work with the new code.
